do_kafka_test.sh

```diff
@@ -1,71 +1,150 @@
 #!/bin/bash
```

Comment:
Should we update the testing.md file to show how to run this and what variables are required? Just trying to run this script after opening in the dev container threw and error of "DOCKER_HOST_IP is not set" which could be avoided with some documentation in that markdown file.

Reply:
Documentation has been added to the README!